lib/Doctrine/DBAL/Types/ConversionException.php

```diff
@@ -120,4 +120,16 @@ public static function conversionFailedSerialization($value, $format, $error)
             $error
         ));
     }
+
+    public static function conversionFailedUnserialization($value, string $format, string $error) : self
+    {
+        $actualType = is_object($value) ? get_class($value) : gettype($value);
+
+        return new self(sprintf(
+            "Could not convert database value '%s' to '%s', as an '%s' error was triggered by the unserialization",
```

Comment:
Since `$value` will very likely be always `string` (unlike the _serialize_ case), I think it doesn't add any value to the error message.

I'd suggest something like this:
```php

"Could not convert database value to '%s' as an error was triggered by the unserialization:  '%s'"
```